src/components/Header/constants.ts

```diff
@@ -52,6 +52,12 @@ const navigations = [
             href: requireEnv("REACT_APP_EXTERNAL_EXPLORER_URI_UNIFRA"),
             isExternal: true,
           },
+          {
+            label: "Dora Explorer",
+            key: "doraBlockExplorer",
+            href: requireEnv("REACT_APP_EXTERNAL_EXPLORER_URI_DORA"),
```

Comment:
Shall we use the URLs from constants here too? Also for unifra

Reply:
Fixed, along with L2 URL. Also removed `requireEnv` import since it's no longer used.